lib/matplotlib/widgets.py

```diff
@@ -3579,7 +3599,8 @@ class PolygonSelector(_SelectorWidget):
     @_api.rename_parameter("3.5", "markerprops", "handle_props")
     @_api.rename_parameter("3.5", "vertex_select_radius", "grab_range")
     def __init__(self, ax, onselect, useblit=False,
-                 props=None, handle_props=None, grab_range=10):
+                 props=None, handle_props=None, grab_range=10, *,
+                 draw_box=False, box_handle_props={}, box_props={}):
```

Comment:
It seems okay here, but you should avoid mutable defaults.

Reply:
Thanks for catching - I'll default these to `None` instead.